src/components/Blog/Blog.styles.tsx

```diff
@@ -1,3 +1,39 @@
+//components
 import styled from 'styled-components';
 
-export const TitlePost = styled.h3``;
+//interface
+import { ImageProps } from '@/interface/interface';
+
+//theme
+import theme from '@/theme/theme';
+
+export const ImagePost = styled.img<ImageProps>`
+    max-width: 40%;
+    min-width: 250px;
```

Comment:
Why px?

Reply:
dlatego, żeby te zdjęcie nie były jakieś mikroskopijne na małych ekranach, bo głupio wygląda to, jak nic nie widać, a zdjęcia będą ważne w postach.